### PR TITLE
Add orbitchat.dev

### PR DESCRIPTION
**Pull request by cppshane**
https://orbitchat.dev

## What kind of PR is this?
**Select only one** *to speed up review/approval; i.e., don't lump an addition and a removal into a single PR.*

- [x] **Adding** something new to awesome-ipfs
- [ ] **Editing** something already listed on awesome-ipfs
- [ ] **Removing** something from awesome-ipfs
- [ ] **Something else** *(if so, please explain in the "Additional details" section below)*

## Pre-submit checklist
**Please confirm ALL of the following** before submitting your PR.

- [x] This PR includes only one addition, removal, or edit.
- [x] I edited the `/data` directory instead of the [README.md](https://github.com/ipfs/awesome-ipfs/blob/master/README.md).
- [x] I reviewed the [content policy](https://github.com/ipfs/awesome-ipfs/blob/master/POLICY.md) and the and the [IPFS Community Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md) to ensure my submission meets the requirements.
- [x] I have followed the [CONTRIBUTING.md guidelines](https://github.com/ipfs/awesome-ipfs/blob/master/CONTRIBUTING.md).

### Is there anything else we should know about this PR?
I built this chat app awhile as an Orbit/IPFS/Angular stack demo. There is also an associated [video](https://youtu.be/O7fxxp_q0es) and [article](https://shaneduffy.io/blog/decentralized-backend-building-an-angular-dapp-with-orbitdb-and-ipfs) that I made. Can I add all of these in this PR? Or should I create two more PRs for each one?

### For new additions: What is your project's *current* size/reach?
It's more of a demo and PoC than anything. It has 17 GitHub stars and gets ~1k unique visitors per month, as per Cloudflare analytics.

### For new additions: What is your project's *potential* size/reach?
I may or may not extend this app further in the future.

### For new additions: How critical is IPFS to your project?

- [x] Essential
- [ ] Somewhat critical
- [ ] Useful, but not critical

### For new additions: What core goal(s) does your project address?

- [ ] Big-data solutions
- [ ] Freedom from corporate/government interference
- [ ] Data integrity
- [ ] Dev tools or other ways to enable developers
- [ ] Disaster resilience/recovery
- [ ] File storage/retrieval
- [ ] File streaming
- [ ] Permanent archiving
- [ ] Self-sovereign identity
- [ ] Platforms that utilize consensus, reputation, or incentivization
- [x] Other: IPFS on web app, IPFS/Orbit integration w/ Angular frontend, PoC of real-time experimental flag IPNS PubSub in the browser

### Is anything about IPFS blocking your project?
No.

### CODE REVIEW

**Review by 2color — CHANGES_REQUESTED** [review]
We've simplified the structure of the repo. Can you please update this PR to include just a link to the source repo in the README.md file?

**Comment by cppshane** [comment]
Updated

File: README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ If you have an idea for an awesome thing to do with IPFS, a good place to ask ab
 - [IPFS-FPS](https://github.com/underscoredLabs/webgl-ipfs-fps) - A completely decentralized first person shooter. Built with Unity, Fleek, Unstoppable Domans and Pinata.
 - [IPGit](https://github.com/meyer1994/ipgit) - Remote Git repository that mirrors your code to IPFS
 - [killcord](https://github.com/nomasters/killcord) - A censorship resistant deadman's switch
+- [orbitchat.dev](https://orbitchat.dev) - Ultra simple chatrooms on the web.
```
Comment (2color) [inline README.md:34]: Can you please link to the repo instead

**Review by 2color — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cppshane) [reply README.md]: Sure, will do

**Comment by 2color** [comment]
Thank you! Have you checked had a chance to look at Helia the new version of the js ipfs implementation?

**Comment by cppshane** [comment]
No, thank you 🙂

Also, I haven't used it yet no, but I've read up on it. I'm working on something else at the moment but once I hop back into IPFS stuff I'll definitely look at implementing it 👍 I've got another web-based IPFS project I've been meaning to get started on.

**Comment by 2color** [comment]
Cool stuff! Be sure to submit it when it's ready.

Also check out https://github.com/libp2p/universal-connectivity

It explores transports

**Pull request merged**